### ISSUE TITLE
Error occur when VM doesn't have Public IP Address

### ISSUE THREAD

**Issue by yokawasa** [yokawasa/azure-ssh-config#2]
The following error occurs when VM doesn't have Public IP Address

```
Traceback (most recent call last):
  File "/usr/local/bin/azuresshconfig", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/azuresshconfig.py", line 314, in main
    ni_info = get_network_interface_info(network_client, ni.id)
  File "/usr/local/lib/python2.7/dist-packages/azuresshconfig.py", line 217, in get_network_interface_info
    public_address_id = ipconfig.public_ip_address.id
AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by yokawasa** [comment]
fixed at version 0.2.1
https://github.com/yokawasa/azure-ssh-config/blob/master/ChangeLog.md#021

**Comment by yokawasa** [comment]
fixed at version 0.2.1
https://github.com/yokawasa/azure-ssh-config/blob/master/ChangeLog.md#021
